error- handlers.js

```diff
@@ -0,0 +1,11 @@
+//postgres errors
+exports.psqlErrorHandler = () => {};
+
+//custom errors
+exports.customErrorHandler = () => {};
+
+//server errors
+exports.serverErrorHandler = (err, req, res, next) => {
+  console.log(err);
+  res.status(500).send({ msg: "Internal server error" });
+};
```

Comment:
Nice job extracting these - technically I would say that this code doesn't belong on this task because you are not using it yet BUT you also have no actual logic in the handlers yet...so I think it's fine 😄 

If you had logic in here to deal with error codes etc I would definitely say remove!